renovate-config/renovate.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+    ":gitSignOff",
+    ":disableDependencyDashboard"
+  ],
+  "ignorePresets": [":dependencyDashboard"],
+  "onboarding": false,
+  "requireConfig": "optional",
+  "platformCommit": true,
+  "autodiscover": false,
+  "enabledManagers": ["tekton", "dockerfile", "rpm"],
```

Comment:
Could you also include the new manager has been added by https://github.com/redhat-appstudio/infra-deployments/pull/4035/files. Note: there is a json syntax issue will be fixed by https://github.com/redhat-appstudio/infra-deployments/pull/4096

Reply:
Added along with fix.